describe/DOType.ts

```diff
@@ -0,0 +1,122 @@
+import { DADescription, describeDA } from "./DADescription.js";
+import { NamingDescription, describeNaming } from "./Naming.js";
+import { SDODescription, describeSDO } from "./SDODescription.js";
+
+function sortRecord(object: Record<string, any>) {
```

Comment:
Is this the first time this function is being used? If this is also needed for sorting extension ns attributes, e.g., might it make sense to factor it out into a shared module?

Reply:
Yes it is. It will be in a shared module eventually, but now it is not needed.